testutil/supplier/fixtures.go

```diff
@@ -0,0 +1,112 @@
+package supplier
+
+import (
+	"testing"
+
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+)
+
+const (
+	testSessionNumber    = 1
+	testBlockHeight      = 1
+	testBlocksPerSession = 4
+	testSessionId        = "mock_session_id"
+)
+
+// SessionsByAppAddress is a map of session fixtures where the key is the
+// application address and the value is the session fixture.
+type SessionsByAppAddress map[string]sessiontypes.Session
+
+// NewSessionFixturesWithPairings creates a map of session fixtures where the key
+// is the application address and the value is the session fixture. App/supplier
+// addresses are expected to be provided in alternating order (as pairs). The same
+// app and/or supplier may be given more than once but only distinct pairs will
+// be added to the session fixtures map.
+func NewSessionFixturesWithPairings(
+	t *testing.T,
+	service *sharedtypes.Service,
+	appAndSupplierAddrPairs ...string,
```

Comment:
I think this warrants a helper struct. It'll make the business logic cleaner. 

If you want to get this over the finish line though, I'd be okay with a `TODO_TECHDEBT/TODO_COMMUNITY`

Reply:
Good call. I found a  low-verbosity solution. :+1: